.eslintrc.js

```diff
@@ -87,5 +87,12 @@ module.exports = {
         '@typescript-eslint/no-use-before-define': 'off',
       },
     },
+    {
+      files: ['test/**/*{.ts,.tsx}'],
+      rules: {
+        'react/prop-types': 'off',
+        'guard-for-in': 'off',
+      },
+    },
```

Comment:
This looks suspicious. Shouldn't this be in the monorepo eslint rule? I don't see why it needs to be defined here.